tests/features/community_content/add_community_content.feature

```diff
@@ -97,7 +98,7 @@ Feature: Add community content
     When I click "Edit" in the "Entity actions" region
     And I press "Publish"
     Then I should see the heading "Sample <content type>"
-    Then the latest version of the "Sample <content type>" <content type> should have the same created date as the last published version
+    Then the published date of the "Sample <content type>" <content type> should not be equal to the created date
```

Comment:
Yes you are right. This should actually check that the publication date of the just published version did not change when publishing the reverted revision.

The general test was indeed coming from a regression test I think, where, when we reverted to an unpublished version, it would re-publish the entity and get a new date. This needs to be updated to check that the entity's publication date is the same as with the previous (published) version